fendermint/vm/interpreter/src/fvm/interpreter.rs

```diff
@@ -213,6 +215,17 @@ where
             .check()
             .map_err(|e| CheckMessageError::InvalidMessage(e.to_string()))?;
 
+        // regardless it is recheck or not, ensure gas fee cap is more than current base fee
+        let base_fee = state.block_gas_tracker().base_fee();
```

Comment:
actually it's filtered by a config in mempool: https://github.com/ethereum/go-ethereum/blob/0983cd789ee1905aedaed96f72793e5af8466f34/core/txpool/txpool.go#L261 and https://github.com/ethereum/go-ethereum/blob/0983cd789ee1905aedaed96f72793e5af8466f34/core/txpool/validation.go#L137C29-L137C35